providers/reported_at/provider_test.go

```diff
@@ -44,5 +44,5 @@ func TestReportedAtFiltering(t *testing.T) {
 	setup(t, RecommendedConfig)
 
 	testProvider.Info(context.Background(), false, "foo")
-	Expect(output.String()).To(MatchRegexp(`time=sometime level=info msg=foo reportedAt=".*go/src/testing/testing.go:\d+`))
+	Expect(output.String()).To(MatchRegexp(`time=sometime level=info msg=foo reportedAt=".*go/.*src/testing/testing.go:\d+`))
```

Comment:
Had to make this change because the tests were failing with the message:

```
Expected
            <string>: time=sometime level=info msg=foo reportedAt="/usr/local/Cellar/go/1.11/libexec/src/testing/testing.go:830"

        to match regular expression
            <string>: time=sometime level=info msg=foo reportedAt=".*go/src/testing/testing.go:\d+
```

Reply:
Nit: Just get rid of the extra `.*go/` part of the regex, it seems pretty useless.
```suggestion
	Expect(output.String()).To(MatchRegexp(`time=sometime level=info msg=foo reportedAt=".*src/testing/testing.go:\d+`))
```